packages/kbn-check-saved-objects-cli/current_mappings.json

```diff
@@ -646,6 +646,9 @@
         "properties": {
           "syncAlerts": {
             "type": "boolean"
+          },
+          "extractObservables": {
+            "type": "boolean"
```

Comment:
I did a quick scan of the PR and couldn't quite tell; are you intending to do searches/aggs on this boolean?

Reply:
no searches/aggs planned at this moment. does that change the mapping if we do?

Reply:
I don't see a flow in which we would want to find all cases that have `extractObservables` enabled or vice versa, unless we wanted some global override, but that's the only use case I would see to need to search on this value